cfme/intelligence/chargeback/assignments.py

```diff
@@ -75,27 +77,21 @@ class Assign(Updateable, Pretty, Navigatable):
     enterprise.computeassign()
 
     """
-    def __init__(self, assign_to=None,
-                 tag_category=None,
-                 docker_labels=None,
-                 selections=None,
-                 appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.assign_to = assign_to
-        self.tag_category = tag_category
-        self.docker_labels = docker_labels
-        self.selections = selections
+    assign_to = attr.ib(default=None)
```

Comment:
Question: Should all of the params be None by default?

Reply:
@rlbabyuk I'm not quite sure, to be honest. This is my first PR. However, as per the values defined in the previous constructor( https://github.com/ManageIQ/integration_tests/blob/69ee8571a9f46c5db3c5b54c46fbc3c9588fdcfe/cfme/intelligence/chargeback/assignments.py#L78 ), I guess it should? However, do let me know what you think and I'll make the necessary changes

Reply:
Required: I assume assign_to should be mandatory as this is a type of assignment, just remove default value